src/SpecialSemanticSignup.php

```diff
@@ -119,7 +119,7 @@ private function userSignup() {
 			$status = $user->sendConfirmationMail();
 
 			if ( !$status->isGood() ) {
-				throw new Exception( wfMsg( 'ses-emailfailed' ) . "\n" . $status->getMessage() );
+				throw new Exception( wfMessage( 'ses-emailfailed' )->escaped() . "\n" . $status->getMessage() );
```

Comment:
Why do you escape here and not above? The 1-to-1 conversion of wfMsg is wfMessage()->text().